lighthouse-core/lib/emulation.js

```diff
@@ -118,8 +120,23 @@ function enableNexus5X(driver) {
   ]);
 }
 
-function enableNetworkThrottling(driver) {
-  return driver.sendCommand('Network.emulateNetworkConditions', TYPICAL_MOBILE_THROTTLING_METRICS);
+/**
+ * @param {Driver} driver
```

Comment:
for type checking in this file we'll have to `require` Driver just to get the type (with an eslint ignore unused above it). Any thoughts on that style? It's pretty ugly but should have no real impact on a node run or browserify (and we obviously use driver so it won't ever be tree shaken if we switch bundling tools).

OTOH there's exposing the major types on `LH` so it could be `{LH.Driver}`, but that has its own ugliness (it's a real JS class, not an interface like the rest of our typings) and I have no idea what the impact on tsc performance will be as we continue to expand the global type graph :)

Reply:
yeah sg, I've used that before 👍